workers/src/main/java/edu/cornell/testenv/testcontext/JUnitTestContext.java

```diff
@@ -0,0 +1,18 @@
+package edu.cornell.testenv.testcontext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JUnitTestContext implements TestEnvContext<String> {
+
+    List<String> classNames;
+
+    public JUnitTestContext(List<String> classNames) {
```

Comment:
Make sure to add documentation and specifications on all your classes, methods, and fields